substrate/utils/frame/omni-bencher/src/main.rs

```diff
@@ -31,7 +31,16 @@ fn main() -> Result<()> {
 
 /// Setup logging with `info` as default level. Can be set via `RUST_LOG` env.
 fn setup_logger() {
-	let env_filter = EnvFilter::try_from_default_env().unwrap_or_else(|_| EnvFilter::new("info"));
+	// Disable these log targets because they are spammy.
+	let unwanted_targets =
+		&["cranelift_codegen", "wasm_cranelift", "wasmtime_jit", "wasmtime_cranelift", "wasm_jit"];
```

Comment:
I think just `wasm` also gets rid of most of the junk.

Reply:
Yes, but that would also filter everything else that has `wasm` prefix, like `wasm-executor` or other future crates :grimacing: